### ISSUE TITLE
Invalid Compound file when saving to stream

### ISSUE THREAD

**Issue by kucint** [ironfede/openmcdf#56]
Hi,
In the currently released version (2.2.1.3), the following code produces an invalid compound file. First 0x200 bytes are set to 0. Do you miss some initialization?
Best!

```
string storageFilepath,=@"C:\temp\storage.cfs"
FileStream storageFile = new FileStream(storageFilepath, FileMode.Create);
CompoundFile compoundFile = new CompoundFile();
compoundFile.Save(storageFile);
compoundFile.Close();
```

**Comment by kucint** [comment]
Hi, I noticed that the Master Version does not have this issue anylonger. If you already corrected it, sorry for bothering.
BTW. when do you plan to release 2.2.1.4?

**Comment by ironfede** [comment]
Released 2.2.1.4.
Thank you for reporting,
Best Regards
